Driver/Driver/Extenison/UIColor+.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  UIColor+.swift
+//  Driver
+//
+//  Created by Fatih Doğan on 22.02.2023.
+//
+
+import UIKit.UIColor
+
+extension UIColor {
+    static var appBlue: UIColor {
+        UIColor(named: "AppBlue")! }
```

Comment:
I don't know yet if we have a rule for force unwrapping, just noting this is a usage

Reply:
we need to force here. We should see any exception like this in the development process. @Racaycah

Reply:
It should either be this or assertionFailure for that, I'm okay with force unwrapping here 😄

Reply:
@Racaycah yeah, You are right. I guess, It is little bit doing over engineering for this project :D @Racaycah